examples/temp.py

```diff
@@ -0,0 +1,10 @@
+import agentops
+from agentops import ActionEvent
+from agent import agent
+
```

Comment:
especially this one. it appears to be a local test file. if you want to leave it in the repo, the best place is `tests/core_manual_tests`